tests/app/dao/test_fact_notification_status_dao.py

```diff
@@ -1,7 +1,7 @@
 from datetime import date, datetime, timedelta
+from unittest import mock
 from uuid import UUID
 
-import mock
```

Comment:
this is odd - why did we need to change where we import mock from? investigate whether this is needed.

Reply:
Looks like unittest.mock is now part of the standard python library:
> mock is now part of the Python standard library, available as [unittest.mock](https://docs.python.org/dev/library/unittest.mock.html) in Python 3.3 onwards.

[Source](https://pypi.org/project/mock/), [Docs](https://docs.python.org/dev/library/unittest.mock.html)

Reply:
yeah, but odd that it works with 3.9 🙃 
But now if you just have `import mock` we get
```
E   ModuleNotFoundError: No module named 'mock'
```
Maybe `import mock` directly was deprecated for a while and finally removed? All good in any case.